cpp/tests/framework/containers/ForwardLinkedList.test.cpp

```
@@ -0,0 +1,81 @@
+// Copyright Robotick Labs
+// SPDX-License-Identifier: Apache-2.0
+
+#include "robotick/framework/containers/ForwardLinkedList.h"
+#include <catch2/catch_all.hpp>
+
+namespace robotick::test
+{
+	struct TestItem
+	{
+		int value = 0;
```

Clang (14.0.6)

[warning] member variable 'value' has public visibility

(misc-non-private-member-variables-in-classes)